Backpack/src/main/res/values-night/backpack.semantic.color.xml

```diff
@@ -18,152 +18,44 @@
-
-	<color name="bpkPrimary">#ff84e9ff</color>
-
-	<color name="__privateChipOnDarkNormalBackground">#ff243346</color>
-
-	<color name="__privateChipOnDarkPressedBackground">#ff002b5b</color>
-
-	<color name="__privateChipOnDarkOnBackground">#ff054184</color>
-
-	<color name="__privateChipDefaultNormalBackground">#ff243346</color>
-
-	<color name="__privateChipDefaultOnBackground">#ff054184</color>
-
-	<color name="__privateChipDisabledBackground">#ff0b121d</color>
-
-	<color name="__privateChipDefaultPressedBackground">#ff002b5b</color>
-
-	<color name="__privateButtonSecondaryPressedBackground">#ff010913</color>
-
-	<color name="__privateButtonLinkNormalForeground">#ff84e9ff</color>
-
-	<color name="__privateButtonLinkOnDarkDisabledForeground">#33ffffff</color>
-
-	<color name="__privateButtonSecondaryNormalBackground">#ff243346</color>
-
-	<color name="__privateButtonPrimaryOnLightPressedBackground">#ff154679</color>
-
-	<color name="__privateButtonFeaturedNormalBackground">#ff84e9ff</color>
-
-	<color name="__privateButtonFeaturedPressedBackground">#ffd1f7ff</color>
-
-	<color name="__privateButtonPrimaryOnLightNormalBackground">#ff05203c</color>
-
-	<color name="__privateButtonPrimaryOnLightDisabledForeground">#33000000</color>
-
-	<color name="__privateButtonSecondaryOnDarkDisabledBackground">#ff0b121d</color>
-
-	<color name="__privateButtonDestructiveNormalForeground">#ffff649c</color>
-
-	<color name="__privateButtonPrimaryOnDarkNormalBackground">#ffffffff</color>
-
-	<color name="__privateButtonPrimaryOnDarkDisabledForeground">#33000000</color>
-
-	<color name="__privateButtonSecondaryOnDarkPressedBackground">#ff010913</color>
-
-	<color name="__privateButtonLinkPressedForeground">#ffd1f7ff</color>
-
-	<color name="__privateButtonPrimaryOnLightDisabledBackground">#ffe0e3e5</color>
-
-	<color name="__privateButtonDestructiveNormalBackground">#ff243346</color>
-
-	<color name="__privateButtonPrimaryNormalBackground">#ff054184</color>
-
-	<color name="__privateButtonDestructivePressedBackground">#ffff649c</color>
-
-	<color name="__privateButtonPrimaryPressedBackground">#ff002b5b</color>
-
-	<color name="__privateButtonPrimaryOnDarkDisabledBackground">#ffe0e3e5</color>
-
-	<color name="__privateButtonLinkOnDarkPressedForeground">#80ffffff</color>
-
-	<color name="__privateButtonLinkOnDarkNormalForeground">#ffffffff</color>
-
-	<color name="__privateButtonDisabledBackground">#ff0b121d</color>
-
-	<color name="__privateButtonSecondaryOnDarkNormalBackground">#ff243346</color>
-
-	<color name="__privateButtonSecondaryOnDarkDisabledForeground">#33ffffff</color>
-
-	<color name="__privateButtonPrimaryOnDarkPressedBackground">#ffc2c9cd</color>
-
-	<color name="__privateSkeletonShimmerStartEnd">#00000000</color>
-
-	<color name="__privateSkeletonShimmerCenter">#33000000</color>
-
- </resources>
+  <!-- THIS FILE IS AUTO GENERATED. DO NOT MODIFY! -->
+
+    <color name="bpkBackground">#ff010913</color>
```

Comment:
the only thing that changed is the order - i actually double checked that (ie didn't sort), but as the compose version is sorted already this makes it easier to use